gitdm_client/README.md

```
@@ -0,0 +1,24 @@
+
+flutter pub get
+flutter run --dart-define=API_BASE_URL=http://127.0.0.1:8000
+```
+
+> **نکته مهم برای اندروید:** پس از تولید پلتفرم، مجوز اینترنت را در `android/app/src/main/AndroidManifest.xml` اضافه کنید:
+>
+> ```xml
+> <uses-permission android:name="android.permission.INTERNET" />
+> ```
+
+## وابستگی‌ها
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)